digitaltwin_client/readme.md

```diff
@@ -2,34 +2,35 @@
 
 This folder contains the headers, source, samples, and internal testing for the Digital Twin Client for the C SDK.
 
-These instructions and samples assume basic familiarity with Digital Twin concepts.  To learn more background information, see [here](https://aka.ms/iotpnpdocs).
+These instructions and samples assume basic familiarity with Digital Twin and Azure IoT Plug and Play concepts.  To learn more, please go [here](https://aka.ms/iotpnpdocs).
 
 ## Getting started
 ### Building
-Instructions on how to build the Digital Twin Client SDK or else retrieve packages for it are available [here](./doc/building_sdk.md).  If you are familiar with building the IoTHub Device SDK, the instructions are almost identical.
+Start by building the Digital Twin Client SDK -- instructions are available [here](./doc/building_sdk.md).  Please use these instructions as they include cmake flags specific to the samples.  On Windows, be sure to clone the repo near your system root to maintain a valid path length.
 
-**Please use [public-preview](https://github.com/Azure/azure-iot-sdk-c/tree/public-preview) branch for your public preview development needs, unless you have been directed to be on public-preview-pnp for a specific purpose.  public-preview-pnp branch is our advance working branch which might contains breaking changes.**
+**Please use the [public-preview](https://github.com/Azure/azure-iot-sdk-c/tree/public-preview) branch unless you have been directed to be on `public-preview-pnp`.  The `public-preview-pnp` branch is our advance working branch which might contain breaking changes.**
 
 ### Exploring samples
+There are two separate, but related, sample directories.
 
-There are two separate but related samples directories.  **It may be easiest to just jump in here for a guided tour of Digital Twin.**
+* For connecting to Iot Hub with a non-resource-constrained device, you will want to use the [digitaltwin\_sample\_device directory](./samples/digitaltwin_sample_device). 
```

Comment:
Please change this "Iot" to have an uppercase T
Like "IoT".

Reply:
also "Azure IoT Hub" ... branding

Reply:
Thanks for catching this.  For other use of Azure,  what about line 3 and 5 with mentions of Digital Twin and Digital Twin Client.  Is Client supposed to be capitalized?  Should it read Azure Digital Twin and Azure IoT Plug and Play?  On line 17, should it be Azure IoT Central?  Thanks :)

Reply:
Don't know about caps, but we should use branding when possible.